projects/plugins/jetpack/_inc/client/recommendations/prompts/prompt-layout/style.scss

```diff
@@ -83,13 +83,17 @@
 	}
 }
 
+.jp-recommendations-question__sidebar-card {
```

Comment:
Related to the layout of the sidebar card - I noticed that just before the mobile breakpoint that the card extends out of the right side of the container.

At the mobile size, the card is not centered - wondering if this would be something to add in this change?
![Screen Shot 2022-03-21 at 3 51 48 PM](https://user-images.githubusercontent.com/18016357/159355726-96abb5f7-f8a6-4b50-b0f9-57d48ca27ab5.png)
![Screen Shot 2022-03-21 at 3 51 58 PM](https://user-images.githubusercontent.com/18016357/159355754-3768d29a-edda-4ea0-b5be-74a52eec5f33.png)